src/App.js

```diff
@@ -9,22 +9,20 @@ const App = () => {
   const [todos, setTodos] = useState([]);
   return (
     <div className="container">
-      <div className="app-wrapper">
-        <div>
-          <Header />
-        </div>
-        <Styleddiv>
-          <TodosList todos={todos} setTodos={setTodos} />
-        </Styleddiv>
-        <Styleddiv2>
-          <Form input={input} setInput={setInput} todos={todos} setTodos={setTodos} />
-        </Styleddiv2>
+      <div>
+        <Header />
       </div>
+      <StyledList>
+        <TodosList todos={todos} setTodos={setTodos} />
+      </StyledList>
+      <StyledForm>
+        <Form input={input} setInput={setInput} todos={todos} setTodos={setTodos} />
```

Comment:
Like I mentioned in my comment above, please don't pass setter functions to components. 

The solution is to wrap your state updater function (setter function) in a callback function and pass that into your child - it's called state up lifting.  
In [THIS ARTICLE](https://www.robinwieruch.de/react-lift-state/) you can read more about this topic. Please note that the syntax is slightly different.